### PR TITLE
Image recs success confetti

### PR DESCRIPTION
**Pull request by sharvaniharan**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt [hunk 1]
```diff
@@ -391,13 +394,35 @@ class ImageRecsFragment : SuggestedEditsItemFragment(), ImageRecsDialog.Callback
...
                 callback().nextPage(this)
                 callback().logSuccess()
             }
         }, duration * durationBoost)
     }
 
+    private fun showConfetti(shouldShowConfetti: Boolean) {
+        binding.successConfettiImage.visibility = if (shouldShowConfetti) VISIBLE else GONE
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            // Change statusBar abd actionBar color
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt:408]: typo: " and "

File: app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt [hunk 2]
```diff
@@ -391,13 +394,35 @@ class ImageRecsFragment : SuggestedEditsItemFragment(), ImageRecsDialog.Callback
...
+        }
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            // Change navigationBar color
+            requireActivity().window.navigationBarColor = if (shouldShowConfetti) ResourceUtil.getThemedColor(requireContext(),
+                    R.attr.image_recs_confetti_background_color) else Color.TRANSPARENT
+        }
+        // Update actionbar menu items
+        requireActivity().window.decorView.findViewById<TextView?>(R.id.menu_help).let {
+            it.visibility = if (shouldShowConfetti) GONE else VISIBLE
+        }
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt:422]: Can use `.apply {   visibility = ... } `

File: app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt [hunk 3]
```diff
@@ -391,13 +394,35 @@ class ImageRecsFragment : SuggestedEditsItemFragment(), ImageRecsDialog.Callback
...
     }
 
+    private fun showConfetti(shouldShowConfetti: Boolean) {
+        binding.successConfettiImage.visibility = if (shouldShowConfetti) VISIBLE else GONE
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            // Change statusBar abd actionBar color
+            requireActivity().window.statusBarColor = if (shouldShowConfetti) ResourceUtil.getThemedColor(requireContext(),
+                    R.attr.image_recs_confetti_background_color) else Color.TRANSPARENT
+            (requireActivity() as AppCompatActivity).supportActionBar!!.setBackgroundDrawable(if (shouldShowConfetti)
+                ColorDrawable(ResourceUtil.getThemedColor(requireContext(), R.attr.image_recs_confetti_background_color)) else null)
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt:412]: Looks like the navigation bar color is not correct. Should it be the same color as the progress bar view background instead?

**Review by cooltey — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt (hunk 3 above)
Comment (sharvaniharan) [reply app/src/main/java/org/wikipedia/suggestededits/ImageRecsFragment.kt:412]: I too thought so... but Robin okayed it this morning... so just want to see what he thinks... or do you feel I should just change to progress bar color?

**Pull request closed without merging**